deployment-templates/compute-engine/client/forseti-instance-client.py

```diff
@@ -133,6 +133,9 @@ def GenerateConfig(context):
 pip install -q --upgrade setuptools wheel
 pip install -q --upgrade -r requirements.txt
 
+# Install instrumentation libs
+pip install .[tracing]
```

Comment:
This is new for me.  But is this relative import the preferred way here?  Would it be better to do specify it as:
```
pip install <module>[tracing]
```
https://stackoverflow.com/a/24640526/2830207